openehr-rm/src/main/java/com/nedap/archie/rm/support/identification/UIDBasedId.java

```diff
@@ -69,4 +69,14 @@ public String getExtension() {
         }
     }
 
+    @Nullable
+    @JsonIgnore
+    public Boolean hasExtension() {
```

Comment:
Shouldn't this be a non-null `boolean`, returning false if getExtension() == null?